routes/services.js

```diff
@@ -0,0 +1,291 @@
+import { PrismaClient } from '../generated/prisma/index.js';
+
+const prisma = new PrismaClient();
+
+class ArticleService {
+  async getAllArticles(query) {
+    const page = parseInt(query.page) || 1; // 페이지
+    const limit = parseInt(query.limit) || 10; // 노출 항목
+    const sort = query.sort || 'recent'; // 정렬 설정
+    const keyword = query.keyword || ''; // 키워드 설정
+    const skip = (page - 1) * limit; // 넘길 항목수
+
+    let orderBy;
+    switch (sort) {
+      case 'resect':
+        orderBy = { createdAt: 'desc' };
+        break;
+      case 'old':
+        orderBy = { createdAt: 'asc' };
+        break;
+      default:
+        orderBy = { createdAt: 'desc' };
+        break;
+    }
```

Comment:
default로 최신순 정렬을 하고싶으신 것 같아요.
불필요하게 조건문 상단에 recent를 먼저 작성할 필요는 없겠죠?
```suggestion
    let orderBy = { createdAt: 'desc' }; // 기본값
    if (sort === 'old') {
      orderBy = { createdAt: 'asc' };
    }
```